kc-cloudtrust-testsuite/kc-cloudtrust-test-tools/src/main/java/io/cloudtrust/keycloak/test/container/KeycloakQuarkusConfiguration.java

```diff
@@ -110,12 +110,12 @@ public KeycloakQuarkusConfigurationBuilder addModuleJar(File jarFile) {
             return this;
         }
 
-        private final Pattern versionPattern = Pattern.compile("^(.+)(-\\d+.\\d+.\\d+)(\\..+)$");
+        private final Pattern versionPattern = Pattern.compile("^(.+)(-\\d+.\\d+.\\d+)(-SNAPSHOT)?(\\..+)$");
```

Comment:
What if you just replace the final `(\\..+)` by `([\\.-].+)` and cancel the change at line 118?

Reply:
I think I did not get what you mean. `([\\.-].+)` is not a valid regex

Reply:
Nevermind, as discussed your solution works